.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!-- Thank you for the contribution! -->
+
+### Why do we need this PR
+
+<!-- Please tell us why is the PR needed -->
+
+### Description of the change
+
+<!-- Please summarize the changes -->
+
+### Related issue number
+
+<!-- For example: "Closes #1234" -->
+
+### How to verify the PR
+
+<!-- Please provide instruction or screenshots on how to verify the PR.-->
+
+### Was the PR tested
```

Comment:
Can we add a checklist here maybe under this heading?

- [ ] I have added >=1 unit test(s) for every new method I have added.
- [ ] I have run `make test` locally and verified all unit tests pass.